### ISSUE TITLE
Design: Update site down for maintenance page

### ISSUE THREAD

**Issue by marlenamellody** [hackforla/tdm-calculator#2258]
### Overview
We need to update the design displayed when the production site is down for maintenance.

### Details
If the production application needs to be taken off-line for maintenance or security fixes, users will reach this page:

<details><summary>Current design when production site is down</summary>

![Image](https://github.com/user-attachments/assets/4ba2796d-8a10-4c3f-bcf0-da30b2bd370f)
</details>

### Action Items
- [x] Create mockups for this page that adheres to the design system
- [x] Review with design. Update if needed based on feedback
- [ ] Review with product and dev. Update if needed based on feedback
- [ ] Once finalized, add before and after images to the [staging deck on this slide](https://docs.google.com/presentation/d/1crZ3IxqA4hAu3qzD7ns93Ieuqjwh6wyEtuX_46cP-fg/edit#slide=id.g3429d2ec911_1_0) 
- [ ] Get Stakeholder sign-off via the stakeholder meeting slide deck.

### Resources/Instructions
- [Figma Design System](https://www.figma.com/design/nD9QK56Mzq7xNSaSUoeGx0/TDM-Calculator?node-id=3337-11981)
Dev issue  #1396 

#### Slides
- [Staging Deck](https://docs.google.com/presentation/d/1crZ3IxqA4hAu3qzD7ns93Ieuqjwh6wyEtuX_46cP-fg)
   - ~[Staging Deck, slide 2258](https://docs.google.com/presentation/d/1crZ3IxqA4hAu3qzD7ns93Ieuqjwh6wyEtuX_46cP-fg/edit?slide=id.g3b88ab30ff0_0_0#slide=id.g3b88ab30ff0_0_0)~
- Release Deck
   - [2026-01-20 Release Deck, slide 2258](https://docs.google.com/presentation/d/1AoHt8euzqHg1sXjXyzZBqIil-bb0RPgF9YDnVRP-gWs/edit?slide=id.g3b91279ffde_0_0#slide=id.g3b91279ffde_0_0)

**Comment by amyzhou06** [comment]
[Figma Page Issue #2258](https://www.figma.com/design/nD9QK56Mzq7xNSaSUoeGx0/TDM-Calculator?node-id=26414-18275&t=sCq311UTxp8zvTfI-0)
Here is the updated page design for when the site is down.

Updated Site Down Page Design:
<img width="557" height="333" alt="Image" src="https://github.com/user-attachments/assets/e79d5cb2-90e7-4001-8451-f1123075d225" />

Notes:

- Updated Icon color to “critical” 

- Updated title text color to “black” to align with other pages 

- Adjusted spacing between elements to match that on other pages: 
            -16px between icon and title text
            -12px between title text and subtitle text
_Is there an established vertical spacing standard between the header and the topmost UI element?_ 

- Changed background vector color to “light gray”
_I followed the design of our current page design for this, but I’m wondering if we want to replace this vector with illustration instead (see references to the right)_

**Comment by ExperimentsInHonesty** [comment]
@amyzhou06 This has been approved by stakeholders.  The dev issue has already been made.  Good work
